src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Baselines/ConfigurationBinder/Get.generated.txt

```diff
@@ -95,7 +95,15 @@ namespace Microsoft.Extensions.Configuration.Binder.SourceGeneration
         public static void BindCore(IConfiguration configuration, ref int[] instance, bool defaultValueIfNotFound, BinderOptions? binderOptions)
         {
             var temp2 = new List<int>();
-            BindCore(configuration, ref temp2, defaultValueIfNotFound: false, binderOptions);
+
+            foreach (IConfigurationSection section in configuration.GetChildren())
```

Comment:
Logic for arrays now inlined. No more `BindCore` to bind to temp `List<T>`.

Reply:
Thank you for calling out the reasoning behind the baseline diffs.